test/basics_datamunger_test.js

```diff
@@ -352,6 +352,67 @@ describe('basics datamunger', function() {
     });
   });
 
+  describe('calculateCarbStats', function() {
+    var wizard = [
+      { type: 'wizard', carbInput: 100, normalTime: '2015-09-01T00:00:00Z' },
+      { type: 'wizard', carbInput: 77, normalTime: '2015-09-01T06:00:00Z' },
+      { type: 'wizard', carbInput: 33, normalTime: '2015-09-01T08:00:00Z' },
+      { type: 'wizard', carbInput: 50, normalTime: '2015-09-02T00:00:00Z' },
+      { type: 'wizard', carbInput: 50, normalTime: '2015-09-03T00:00:00Z' }
+    ];
+
+    it('should be a function', function() {
+      assert.isFunction(dm.calculateCarbStats);
+    });
+
+    describe('averageDailyCarbs', function() {
+      it('should calculate average daily carbs', function() {
+        var basicsData = {
+          data: {
+            wizard: { data: wizard }
+          },
+          dateRange: [
+            '2015-09-01T00:00:00.000Z',
+            '2015-09-03T00:00:00.000Z'
+          ],
+          days: [{
+            date: '2015-09-01',
+            type: 'past',
+          }, {
+            date: '2015-09-02',
+            type: 'past',
+          }, {
+            date: '2015-09-03',
+            type: 'mostRecent',
+          }]
+        };
+
+        expect(dm.calculateCarbStats(basicsData).averageDailyCarbs).to.equal(130);
+      });
+
+      it('should exclude any carbs falling outside date range', function() {
```

Comment:
This test, like the code above referencing a `start` and `end` isn't necessary since the `wizard` source for the carbs data is point-in-time.